go-controller/pkg/util/node_annotations.go

```diff
@@ -591,3 +596,19 @@ func GetNodeID(node *kapi.Node) int {
 func NodeIDAnnotationChanged(oldNode, newNode *corev1.Node) bool {
 	return oldNode.Annotations[ovnNodeID] != newNode.Annotations[ovnNodeID]
 }
+
+// SetNodeZone sets the node's zone in the 'ovnNodeZoneName' node annotation.
+func SetNodeZone(nodeAnnotator kube.Annotator, zoneName string) error {
+	return nodeAnnotator.Set(ovnNodeZoneName, zoneName)
+}
+
+// GetNodeZone returns the zone of the node set in the 'ovnNodeZoneName' node annotation.
+// If the annotation is not set, it returns the 'default' zone name.
+func GetNodeZone(node *kapi.Node) string {
+	zoneName, ok := node.Annotations[ovnNodeZoneName]
+	if !ok {
+		return types.OvnDefaultZone
+	}
```

Comment:
I think so, particularly in the OCP case where each node is a zone.

If ovnkube-ncm/master starts first, it will consider the its own single zone node as remote until ovnkube node sets this annotation.

Reply:
These comment lost relevance when I saw we were able to handle nodes changing zones dynamically in a later commit. 

But the point I was trying to make is if it would be better to do nothing until we are aware of what zone we are in for certain versus assuming the global zone.

Reply:
Right.  Please see the commit 9.  Where the zone of the node is passed as an argument during start up and it matches to the db zone to be sure both are the same

Reply:
I guess no action to be taken here